android/ui/src/main/java/com/trafi/ui/theme/Spacing.kt

```diff
@@ -21,5 +21,10 @@ object Spacing {
 
 @Immutable
 data class MaasSpacing(
-    val globalMargin: Dp = SpacingScale.Default.GlobalMargin
+    val globalMargin: Dp = SpacingScale.Default.GlobalMargin,
+    val badgeSpacer: Dp = SpacingScale.xxs,
```

Comment:
I think badge spacer should only be part of BadgeConstants. 
MaasSpacing should not know about any component 🤔
Or I missed something?

Reply:
Yes, I completely agree 👍

Reply:
General rules of thumb -
1. do not put configurable values into these global theme values unless necessary
1. to judge if necessary, consider whether this is something we theme globally per rollout in our own whitelabel apps